test/src/test/java/hudson/model/ItemGroupMixInTest.java

```diff
@@ -47,4 +65,133 @@
         assertEquals("valid", items.iterator().next().getName());
     }
 
+  /**
+   * This test unit makes sure that if part of the config.xml file is
+   * deleted it will still load everything else inside the folder.
+   * The test unit expects an IOException is thrown, and the one failed
+   * job fails to load.
+   */
+  @Issue("JENKINS-22811")
```

Comment:
BTW for the future please use 4-space indentation to be consistent with other Jenkins code.